ansible/all-in-one/deploy.yaml

```diff
@@ -79,6 +79,8 @@
     xsnippet_spa_https_redirect: true
     xsnippet_spa_ssl_cert: /home/xsnippet/xsnippet_spa_ssl.cert
     xsnippet_spa_ssl_key: /home/xsnippet/xsnippet_spa_ssl.key
+    # limit API requests per IP address at nginx level
+    xsnippet_api_rate_limits: true
```

Comment:
(thought aside) Some time ago when I was working on Ansible playbook for personal usage I found that making everything configurable increase support cost and decrease readability. So I want to ask: do we plan to deploy somewhere XSnippet without rate limit? If not, maybe hardcode these values and and do not bother to make it enabled/disabled via some flag? What do you think?